packages/server/src/environment.js

```diff
@@ -45,6 +45,7 @@ module.exports = {
   INTERNAL_API_KEY: process.env.INTERNAL_API_KEY,
   MULTI_TENANCY: process.env.MULTI_TENANCY,
   HTTP_MIGRATIONS: process.env.HTTP_MIGRATIONS,
+  API_RATE_LIMITING: process.env.API_RATE_LIMITING,
```

Comment:
Similar to above, would be nice if we mentioned `API_LIMITING_REQUESTS_PER_MINUTE` or `API_LIMITING_REQUESTS_PER_SECOND` as suggested